configuration/concepts/misc.csv

```diff
@@ -0,0 +1,4 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1000
+81f330b5-3f10-11e4-adec-0800271c1b75,,,Document,Document,,Misc,Text,,,,,,,,,,,
```

Comment:
Isn't this concept already created via inheriting the OpenMRS Config Haiti?
See here https://github.com/mekomsolutions/openmrs-config-haiti/blob/master/configuration/concepts/misc.csv#L34

Reply:
Something is wrong here, when i do mvn clean install on this bahmni-distro-easydoc i doesn't bring these **Patient Document** concept.

What make me think that this error is from the **Patient Document** concept is that I have created them in openmrs administration and it works well.